INSTALL/Install_configure_postgreSQL_CENTOS7.md

```diff
@@ -0,0 +1,126 @@
+Following steps will let you install, configure postgreSQL server and create a
+`haas` database for development or production version of HaaS on a 
+Centos - 7 server
+
+
+
+## Part 1: 
+Install PostgreSQL server. Initialize the system. Configure PostgreSQL to allow password authentication.
+
+**1. Install the requisite packages on your server.**
+
+```
+sudo sudo yum install postgresql-server postgresql-contrib -y
+```
+
+**2. Initialize postgresql:**
+
+```
+sudo postgresql-setup initdb
+```
+
+**3. Replace the term `ident` from following lines**
+in file `/var/lib/pgsql/data/pg_hba.conf` with `md5`
+
+**Before:**
+
+```
+# "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+host    all             all             127.0.0.1/32            ident
+host    all             all             ::1/128                 ident
+#host    replication     postgres        127.0.0.1/32            ident
+#host    replication     postgres        ::1/128                 ident
+```
+
+**After:**
+
+```
+# "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+host    all             all             127.0.0.1/32            md5
+host    all             all             ::1/128                 md5
+#host    replication     postgres        127.0.0.1/32            ident
+#host    replication     postgres        ::1/128                 ident
+```
+
+**4. Start postgresql service**
+
+```
+$ sudo systemctl start postgresql
+$ sudo systemctl enable postgresql
+```
+
+##Part 2: 
+Setup system user and database role for HaaS.Create database `haas` owned by user `haas`.
+
+**5. Create a system user haas:**
```

Comment:
I still think that this should just refer users to the system user that should already have been created. Since this document is part of a semi-sequential series of steps it does not make sense to instruct users to create a system user twice.

I would be fine with including a note that indicated that this should only be done if a system user has not already been created.